aiohttp/resolver.py

```diff
@@ -26,6 +27,15 @@ def __init__(self, loop=None):
         self._loop = loop
 
     async def resolve(self, host, port=0, family=socket.AF_INET):
```

Comment:
1. Since it is private API, I suggest changing default value for family from AF_INET to AF_UNSPEC.
2. I would prefer IPv6 to be the first if AF_UNSPEC requested.

Reply:
+1 for `AF_UNSPEC`